src/callbacks_step/analysis.jl

```diff
@@ -606,6 +619,26 @@ function analyze_integrals(analysis_integrals::Tuple{}, io, du, u, t, semi)
     nothing
 end
 
+# Iterate over tuples of analysis integrals in a type-stable way using "lispy tuple programming".
+function analyze_pointwise(analysis_quantities::NTuple{N, Any}, du, u, t,
+                           semi, iter) where {N}
+
+    # Extract the first analysis integral and process it; keep the remaining to be processed later
```

Comment:
Same as above. These are not integrals.

```suggestion
    # Extract the first pointwise analysis quantity and process it; keep the remaining to be processed later
```